pkg/cmd/runtime/plugins/k8s/k8s_runtime_test.go

```diff
@@ -69,6 +72,13 @@ spec:
 `
 
 func TestK8sRuntime(t *testing.T) {
+	clientConf := "~/.kube/config"
+	_, err := os.Stat(strings.Replace(clientConf, "~/", os.Getenv("HOME")+"/", 1))
+	if err != nil {
+		fmt.Printf("K8s runtime test skipped because no [%s] %s", clientConf, err)
```

Comment:
s/fmt.Printf/t.Logf/

Reply:
t.Logf + t.Skip() is the same with t.Skipf()
But I have tried without ~/.kube/config, no output at all.  
If I use fmt.Printf, run go test -v ./..., I can see the output.

Reply:
`t.Logf` or `t.Skipf` will print the output when run `go test -v`

Reply:
I have tried, but nothing output. like this:
PASS
ok  	openpitrix.io/openpitrix/pkg/cmd/runtime/plugins/k8s	0.076s
PASS
ok  	openpitrix.io/openpitrix/pkg/cmd/runtime/plugins/qingcloud	0.033s


And go test -v , config.test failed